package.json

```diff
@@ -6,9 +6,9 @@
   "author": "Andrew Kirilenko, Yury Yantsevich & Iron.io, Inc",
   "main": "./lib/client",
   "dependencies": {
-    "pkginfo": "0.2.3",
-    "underscore": "1.3.3",
-    "iron_core": ">=0.2.3"
+    "pkginfo": "0.3.0",
+    "underscore": "1.4.4",
```

Comment:
Current production version of `underscore.js` is 1.7.0. Is there any reason to update it to v1.4.4 instead of upstream?